### ISSUE TITLE
Command not found: _pipenv

### ISSUE THREAD

**Issue by thepreacher** [marlonrichert/zsh-snap#151]
```
znap function _pipenv pipenv            'eval "$( pipenv --completion )"'
compdef       _pipenv pipenv
```
The code above is from the sample `.zshrc` and it used to work before for pipenv completions. However after `znap pull` today - there were a few updates - when I type `pipenv ` and ⏎, I get the message `command not found: _pipenv`.

**Comment by thepreacher** [comment]
I unfortunately can't provide a way to reproduce the issue since I just installed `pipenv` using `pipx` and it's always worked. However reading the `pipenv` documentation this evening, I noticed a change in the command used to generate the completions. It used to be `eval "$( pipenv --completion )"` but now they provide `eval "$(_PIPENV_COMPLETE=zsh_source pipenv)"`. 

I'm not sure if that affects the current directive for how to setup `pipenv` completion using `zsh-znap`? My pipenv version is `Release v2021.11.23`
